### PR TITLE
Journalist can create an article and styling

### PR DESCRIPTION
**Pull request by nevroje**
#### Description

Create method added in staff_content controller
added form to staff view

#### What I have learned working on this feature:
How to create classes/divs in haml, and to add my own css in the tailwind file for example a new font etc. 

#### Screenshots: 
image

### CODE REVIEW

File: features/journalist_can_create_an_article.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
+Feature: Jorunalist can create an article
```
Comment (LauraRe) [inline features/journalist_can_create_an_article.feature]: typo in Journalist

File: features/journalist_can_create_an_article.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+			| entertainment |
+			| global        |
+			| politics      |
+		Given the following users exist
+			| email           | role    		 	  |
+			| amanda@mail.com | journalist			|
+			| viktor@mail.com | registered	    |
+
+	Scenario: Journalist can create an article
+		Given I am logged in as "amanda@mail.com"
```
Comment (LauraRe) [inline features/journalist_can_create_an_article.feature]: line 22-25 in background?

File: features/journalist_can_create_an_article.feature [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		Then I should see "Welcome to the staff page"
+		And I fill in "title" with "Title goes here"
+		And I fill in "content" with "Content goes here"
+		And I select "Free" from "status-select"
+		And I select "Sports" from "categories-select"
+		And I click "Save Article"
+		Then I should see "Article was successfully created."
+
+	Scenario: Journalist cant create and artcile [SadPath]
+		Given I am logged in as "amanda@mail.com"
```
Comment (LauraRe) [inline features/journalist_can_create_an_article.feature]: line 34-37 in background?

File: features/journalist_can_create_an_article.feature (hunk 2 above)
Comment (nevroje) [reply features/journalist_can_create_an_article.feature]: fixed

File: features/journalist_can_create_an_article.feature (hunk 3 above)
Comment (nevroje) [reply features/journalist_can_create_an_article.feature]: fixed

File: features/journalist_can_create_an_article.feature (hunk 1 above)
Comment (nevroje) [reply features/journalist_can_create_an_article.feature]: fixed

File: features/journalist_can_create_an_article.feature [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	Background:
+		Given the following categories exist
+			| name          |
+			| sports        |
+			| tech          |
+			| local         |
+			| entertainment |
+			| global        |
+			| politics      |
+		Given the following users exist
```
Comment (LauraRe) [inline features/journalist_can_create_an_article.feature]: Change some 'given' to 'and'

File: features/journalist_can_create_an_article.feature [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+		When I click on "Employee"
+		Then I should see "Welcome to the staff page"
+		And I fill in "title" with "Title goes here"
+		And I fill in "content" with "Content goes here"
+		And I select "Free" from "status-select"
+		And I select "Sports" from "categories-select"
+		And I click "Save Article"
+		Then I should see "Article was successfully created."
+
+	Scenario: Journalist cant create and artcile [SadPath]
```
Comment (aleciabenjamin) [inline features/journalist_can_create_an_article.feature]: typo article

File: app/controllers/articles_controller.rb [hunk 6]
```diff
@@ -4,6 +4,9 @@ def index
 		@articles = Article.all
 	end
 
+	def create
```
Comment (LauraRe) [inline app/controllers/articles_controller.rb]: do we need this?

Comment (nevroje) [reply app/controllers/articles_controller.rb]: We added this and then it worked

**Pull request merged**